packages/platform-sdk-generator/src/generateResource.ts

```diff
@@ -40,8 +40,8 @@ export async function generateResource(resource: Resource, nsPath: string) {
   await writeCode(
     filePath,
     `${copyright}\n\n
-        import type { ClientContext as $ClientContext, OmniMethod as $OmniMethod } from "@osdk/shared.net";
-        import { omniFetch as $omniFetch } from "@osdk/shared.net";
+        import type { SharedClient as $Client, SharedClientContext as $ClientContext, FoundryPlatformMethod as $FoundryPlatformMethod } from "@osdk/shared.net";
```

Comment:
We now support being passed a "client" (something that has a property of the symbol `symbolClientContext` or a `SharedClientContext`